reup/revive/views.py

```diff
@@ -7,11 +8,15 @@ def index(request):
     ctx = {}
 
     try:
-        if 'id' in request.POST:
-            ctx['document'] = Document.objects.get(old_id=request.POST['id'])
+        if 'input' in request.POST:
+            idMatch = re.search('\d+', request.POST['input'])
```

Comment:
per Python conventions, please call the variable `id_match`, and use a raw string for the regexp, even if it's not strictly needed (`re.search(r'\d+', ...)`)

Reply:
Updated.